src/styles/link-card-styles.css

```diff
@@ -0,0 +1,7 @@
+.link-card {
+  text-decoration: none;
+  color: #007eb9;
```

Comment:
Let's please start using Amplify UI variables when and where it makes sense. For example, instead of #007eb9, we can use var(--amplify-colors-font-primary). Additionally, please be sure to add :focus and :hover styles.

Reply:
Yes will add them!